crates/legacy/src/service.rs

```diff
@@ -923,98 +912,61 @@ impl<Types: NodeType> ProxyGlobalState<Types> {
-
-            let timeout_after = Instant::now() + self.max_api_waiting_time;
-            let check_duration = self.max_api_waiting_time / 10;
-
-            let response_received = loop {
-                match timeout(check_duration, vid_receiver.write().await.get()).await {
-                    Err(_toe) => {
-                        if Instant::now() >= timeout_after {
-                            tracing::warn!("Couldn't get vid commitment in time for block {id}",);
-                            {
-                                // we can't keep up with this block size, reduce max block size
-                                self.global_state
-                                    .write_arc()
-                                    .await
-                                    .block_size_limits
-                                    .decrement_block_size();
-                            }
-                            break Err(ClaimBlockHeaderInputError::CouldNotGetVidInTime);
-                        }
-                        continue;
-                    }
-                    Ok(recv_attempt) => {
-                        if recv_attempt.is_err() {
-                            tracing::error!(
-                                "Channel closed while getting vid commitment for block {id}",
-                            );
-                        }
-                        break recv_attempt
-                            .map_err(ClaimBlockHeaderInputError::WaitAndKeepGetError);
-                    }
-                }
-            };
-
-            tracing::info!("Got vid commitment for block {id}",);
-
-            // We got VID in time with margin left.
-            // Maybe we can handle bigger blocks?
-            if timeout_after.duration_since(Instant::now())
-                > self.max_api_waiting_time / VID_RESPONSE_TARGET_MARGIN_DIVISOR
-            {
-                // Increase max block size
-                self.global_state
-                    .write_arc()
-                    .await
-                    .block_size_limits
-                    .try_increment_block_size(truncated);
-            }
+            // tracing::info!("Waiting for vid commitment for block {id}");
+
+            // let timeout_after = Instant::now() + self.max_api_waiting_time;
+
+            // tracing::info!("Got vid commitment for block {id}",);
+
+            // TODO: Should this still be here?
+            // // We got VID in time with margin left.
+            // // Maybe we can handle bigger blocks?
+            // if timeout_after.duration_since(Instant::now())
+            //     > self.max_api_waiting_time / VID_RESPONSE_TARGET_MARGIN_DIVISOR
+            // {
+            //     // Increase max block size
+            //     self.global_state
+            //         .write_arc()
+            //         .await
+            //         .block_size_limits
+            //         .try_increment_block_size(truncated);
+            // }
```

Comment:
Nit: Can we delete this commented-out part if no longer needed?

Reply:
cc @pls148